src/redist/targets/GenerateMSIs.targets

```diff
@@ -70,7 +71,7 @@
           PropertyName="SdkInstallerUpgradeCode" />
     </GenerateGuidFromName>
 
-    <GenerateGuidFromName Name="$(CombinedFrameworkSdkHostMSIInstallerFile)">
+    <GenerateGuidFromName Name="$(SdkBundleInstallerOutputGuidString)">
```

Comment:
@johnbeisner I'm not familiar with this task. Just want to make sure that this is generating a UUID3 guid.

Reply:
@joeloff 
"...does this take the x86/x64 architecture into account so that x86 and x64 have different GUIDs?" - yes, the upgrade code is based on: "an arbitrary string + the SDK product-band [3.0.1] + the bitness"

Reply:
Going way back in time, this is apparently a **Version 5** GUID: https://github.com/brthor/cli/blob/0d37da4d0d63d4f1ff1c5b92fab0b55ce690acfd/build_projects/shared-build-targets-utils/Utils/Utils.cs#L41-L42. It looks like that comment was lost in some refactoring: https://github.com/dotnet/cli/pull/3696/files. (This is also the algorithm we have in Core-Setup, since it's a fork of CLI.)

Reply:
I've put the long lost comment back in the code.

Reply:
My understanding is that Version 5 UUID uses SHA1, and Version 3 UUID uses MD5, but they're doing about the same thing. So Version 5 is also ok (and probably preferable). @joeloff is that correct?

Reply:
Yup, the important thing is to ensure that we generate stable GUIDs for the same set of inputs